Userland/Libraries/LibGUI/FileSystemModel.h

```diff
@@ -81,9 +81,9 @@ class FileSystemModel
 
         FileSystemModel& m_model;
 
-        Node* parent { nullptr };
-        NonnullOwnPtrVector<Node> children;
-        bool has_traversed { false };
+        Node* m_parent { nullptr };
+        NonnullOwnPtrVector<Node> m_children;
+        bool m_has_traversed { false };
```

Comment:
The important changes would be a lot easier to read if we did these renames in a separate commit :)

Reply:
Gotchu, will move the renames to a new commit.